plugins/GSdx/Renderers/DX11/GSTexture11.h

```diff
@@ -44,6 +44,7 @@ class GSTexture11 : public GSTexture
 	bool Map(GSMap& m, const GSVector4i* r = NULL, int layer = 0);
 	void Unmap();
 	bool Save(const std::string& fn, bool dds = false);
+	bool Equal(GSTexture11 *tex);
```

Comment:
Can it be `GSTexture11* tex` instead of `GSTexture11 *tex`? I think it is more in line with rest of coding style.

Reply:
Ah, my personal coding style is leaking. Thanks for catching that.